chunky/src/java/se/llbit/chunky/block/Water.java

```diff
@@ -21,9 +21,6 @@
   public static final Water INSTANCE = new Water(0);
 
   public static final double TOP_BLOCK_GAP = 0.125;
-  
-  // Used only as starting material when camera is submerged.
```

Comment:
I wonder if this was important. Like is there something that is broken now, e.g. if the camera is submerged?